cluster/README.md

```diff
@@ -0,0 +1,258 @@
+# Kubernetes Cluster and Image Management
+
+As you might notice, playing games is fairly slow.  One way to speed up playing
```

Comment:
Now without a segue from the previous paragraph ;)  Maybe rephrase

Reply:
Removed segue